README.md

```diff
@@ -1 +1,96 @@
-# presto-alibaba-maxcompute-connector
+# MaxCompute Connector
+
+The MaxCompute connector enables direct querying and processing of data stored in the MaxCompute data warehouse. This connector is particularly useful for implementing data
+integration and analysis between MaxCompute and other systems, such as Hive.
+
+## MaxCompute Connector Setup
+
+### Prerequisites
+
+Before using the MaxCompute connector, please ensure the following conditions are met:
+
+- Access permissions: Ensure you have valid access rights to a MaxCompute project and that you have obtained the necessary authentication credentials.
+- Network setup: For stable data transmission, the use of Aliyun VPC network is recommended. Check out
+  the [MaxCompute Network Configuration Guide](https://help.aliyun.com/zh/maxcompute/user-guide/network-connection-process) for more information.
+- Resource group configuration: Possession of an exclusive Tunnel resource group is required (although MaxCompute currently allows access to shared resource groups, this could
```

Comment:
```suggestion
- Resource group configuration: Possession of an exclusive Tunnel resource group is required. Note: MaxCompute currently allows access to shared resource groups, but this could
```